meteor/lib/lib.ts

```diff
@@ -49,6 +49,23 @@ export function getHash(str: string): string {
 		.digest('base64')
 		.replace(/[\+\/\=]/g, '_') // remove +/= from strings, because they cause troubles
 }
+/** Returns a string that'll change whenever the content of the object changes (excluding ordering of properties) */
```

Comment:
As this is written it seems like the string returned will change when the content of the argument object changes The code returns a string literal which will never change.